src/Home.js

```diff
@@ -26,13 +27,23 @@ function Home () {
       }),
   })
   
-  const allGirls = data && ('pages' in data) ? data.pages.flat() : []
+  const allGirls = data && ('pages' in data) ? data.pages.flat() : [];
+
+  const valueDir = (dir) => {
+    return (dir == 'left') ? 0 : (dir == 'right') ? 1 : (dir == 'up') ? 2 : 99;
+  }
   
   const swiped = (direction, id) => {
     const girl = allGirls.find((item) => item.id === id);
     console.log('swiping ', girl);
     !swipedGirls.current.includes(id) && swipedGirls.current.push(id)
     setLastDirection(direction);
+    sendEvent({
+      category: 'interaction',
+      action: 'swiping',
+      label: girl.link_display,
```

Comment:
perhaps it's better to send the actual Tweet ID or picture ID? So that later we can group by the label along with other actions

Reply:
ahh, since all label are using girl.link_display, we can just use this :)